src/common/lspLanguageFeatures.ts

```diff
@@ -105,7 +106,10 @@ export class DiagnosticsAdapter<T extends ILanguageWorkerWithDiagnostics> {
 			.then((worker) => {
 				return worker.doValidation(resource.toString());
 			})
-			.then((diagnostics) => {
+			.then((allDiagnostics) => {
+				const diagnostics = allDiagnostics
+						.filter(diagnostic => this._diagnosticCodesToIgnore.indexOf(diagnostic.code) === -1);
```

Comment:
can you use !includes?

Reply:
Ended up not using because it was meant to use as it is due to being consistent with the code from the Typescript language, for example.